migrations/20210421135937-migrate-platform-tip-transaction-group.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+/**
+ * Migrate all entries (1198) missed by `migrations/20201016085526-add-platform-contribution-transaction-group.js`,
```

Comment:
Used the query dropped in https://github.com/opencollective/opencollective-api/pull/4716#pullrequestreview-512415868 to migrate the platform tips without `PlatformTipForTransactionGroup` as it seems to catch more entries than the fixed version of the query joining on [`createdAt`](https://github.com/opencollective/opencollective-api/blob/5654a97eb4417b80ebb90aca07931f0f8961678d/migrations/20201016085526-add-platform-contribution-transaction-group.js).